XenAdmin/TabPages/NICPage.resx

```diff
@@ -212,14 +212,17 @@
   <data name="ColumnFCoECapable.Width" type="System.Int32, mscorlib">
     <value>105</value>
   </data>
-  <metadata name="SriovSupportedColumn.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
+  <metadata name="ColumnSriovSupported.UserAddedColumn" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
     <value>True</value>
   </metadata>
-  <data name="SriovSupportedColumn.HeaderText" xml:space="preserve">
+  <data name="ColumnSriovSupported.HeaderText" xml:space="preserve">
     <value>SR-IOV supported</value>
   </data>
-  <data name="SriovSupportedColumn.Width" type="System.Int32, mscorlib">
-    <value>114</value>
+  <data name="ColumnSriovSupported.MinimumWidth" type="System.Int32, mscorlib">
+    <value>150</value>

```

Comment:
The correct way to fix this is by disabling the columns' height resizing mode, which fixes the problem for all columns. Setting a minimum width to this column will not prevent the problem from occurring again if a new column is added to the gridview or even now in a different language. The minimum width does not need to change, as long as it is big enough to prevent the column from disappearing completely and allow the user to easily expand it.